docs/PINOUT.MD

```diff
@@ -0,0 +1,13 @@
+# PINOUT for digital temperature sensor DS18B20
+                                                NUCLEO PIN                SENSOR PIN
+                                                   PC8  ------------------->  DQ
+                                                   +5V  ------------------->  VDD
+                                                   GND  ------------------->  GND
+To correct work temperature sensor should be connected also resistor pull up between pin VDD and DQ.
```

Comment:
Pull up resistor should be connected between pin VDD and DQ to assure correct work of temperature sensor.

Reply:
fixed